feat(content): New Alien Faction: The Gegno (#6610)
endless-sky/endless-sky: data/gegno/gegno intro missions.txt

			`	You aren't sure how to begin a conversation, but start by trying to greet the alien in a formal manner. Awkward silence ensues, and the alien looks you up and down in an almost unenthused manner. They emit a very deep grumble, and with a loud snort, reach into a crevice in their armor to grab something. Pulling out what looks to be some sort of compass or map, they point to a location on it, gesture to themselves, then lower the device slowly to you. It seems they want to be taken to one of their species' planets.`
			`	You hesitantly take the device and turn to make your way to your ship. There doesn't feel to be any emergency to the situation, but you still feel uncomfortable. Before heading out, you take one last glimpse of the enormous creature in the background. Its head is very reptilian, close to dragon-like, with numerous eyes and a mane of spikes. A few notably small appendages are found at either end of it, almost like a salamander, but its surface is definitely scaly. The various signatures on your radar earlier must have been others of their kind. Glaring at you, then at the creature, the alien utters a single intelligible word in a very deep voice:`

> 			`	"Voordt."

Reviewer: ```suggestion
			`	"Voordt."`
```

mission "Gegno Vi: First Contact"
	landing
	source "Var Oti"
	on offer
		conversation		
			branch "second encounter"
				has "Giaru Gegno: Quarg Contact: offered"
			branch "return encounter"
				has "Gegno Vi: First Contact: deferred"
				

> 			`As you descend towards the surface of this seemingly uninhabited world, your radar systems start to pick up several very faint signatures below in every direction. None of the signatures change their behavior as you continue, but you decide keep your ship a few thousand meters above the surface while staying wary in alien territory. You take the time to carefully look around, but there doesn't seem to be anything of note besides the cracked rock formations and valleys of sand.`

Reviewer: ```suggestion
			`As you descend towards the surface of this seemingly uninhabited world, your radar systems start to pick up several faint signatures below in every direction. None of the signatures change their behavior as you continue, but you decide to keep your ship a few thousand meters above the surface while staying wary in alien territory. You take the time to carefully look around, but there doesn't seem to be anything of note besides the cracked rock formations and valleys of sand.`
```

			`Suddenly, your radar systems pick up a larger signature further up ahead on the horizon, and in the distance is an odd bulky rock-like structure. Once you get close enough, the structure begins to resemble ships you've seen flying around nearby systems, although it is half sunken into the sand.`
			choice

> 				`	Land near the ship and investigate.`

Reviewer: ```suggestion
				`	(Land near the ship and investigate.)`
```

					goto land

> 				`	Continue to search the surface.`

Reviewer: ```suggestion
				`	(Continue to search the surface.)`
```

					goto search
			label land
			`	Still keeping your guard up, you carefully land your ship nearby. Most of the faint radar signatures from earlier have disappeared out of range, so it seems there's nothing coming after you at the moment. Exiting your ship in an environment suit with your sidearm on hand, you walk towards the alien ship while remaining alert.`

> 			`	On closer inspection, the alien ship appears a bit larger than you thought it was, around the size of a Firebird. The dark-brown hull looks to be made from materials that you aren't familiar of, and it has several metallic-looking parts both hatched in crevices and protruding outward. Oddly enough, it seems perfectly intact, but the way it is angled into the sand suggests that the ship has been here for some time. There doesn't seem to be any activity around it, and you aren't picking up any life signs nearby. Leaving the site, you decide to head back to your ship and take another look around the area.`

Reviewer: ```suggestion
			`	On closer inspection, the alien ship appears a bit larger than you thought it was - around the size of a Firebird. The dark-brown hull looks to be made from materials that you aren't familiar with, and it has several metallic-looking parts both hatched in crevices and protruding outward. Oddly enough, it seems perfectly intact, but the way it is angled into the sand suggests that the ship has been here for some time. There doesn't seem to be any activity around it, and you aren't picking up any life signs nearby. Leaving the site, you decide to head back to your ship and take another look around the area.`
```

				
			label search
			`	Continuing on your journey, a vague radar signature similar to the ones you saw earlier, though more erratic, appears a bit further out. As you approach it, a large arch-like structure becomes visible in the distance, but disappears shortly after. You set your ship down close by and ready yourself as you exit it once again to investigate.`
			`	Much to your surprise, there is a large humanoid figure standing where the signature originated from. You were sure your radar scans showed no life form signature, but standing in front of you is a large, bulky being cladded in what looks like a form of metallic armor, wielding some type of spear-like weapon. The figure moves around attentively as if it is intently focused on its surroundings, but appears to take no interest in your presence. Unlike other aliens you have met in the galaxy, this encounter feels very odd in that there is no immediate interaction with you.`
			choice

> 				`	Approach the alien.`

Reviewer: ```suggestion
				`	(Approach the alien.)`
```

> 				`	Turn back to your ship.`

Reviewer: ```suggestion
				`	(Turn back to your ship.)`
```

> 			`	All of a sudden, the ground starts to vibrate, rapidly increasing in intensity with every second. You take out your sidearm in a panic, but as you look back up at the alien figure, you notice that they. too, enter a defensive stance- but not towards you. A large maw bursts open in the sand to your left, and out of it comes a massive creature covered in spikes and scales, hundreds of times the size of you. Taking aim with your sidearm, you barely have time to react as it plummets back into the sand over beyond your right, but get a quick glimpse of the alien creature riding it down into the desert as they impale it with their weapon. Unsure of what comes next during the omnious silence, you slowly start to back your way towards your ship.`

Reviewer: ```suggestion
			`	All of a sudden, the ground starts to vibrate, rapidly increasing in intensity with every second. You take out your sidearm in a panic, but as you look back up at the alien figure, you notice that they too enter a defensive stance - but not towards you. A large maw bursts open in the sand to your left, and out of it comes a massive creature covered in spikes and scales, hundreds of times the size of you. Taking aim with your sidearm, you barely have time to react as it plummets back into the sand over beyond your right, but you get a quick glimpse of the alien creature riding it down into the desert as they impale it with their weapon. Unsure of what comes next during the omnious silence, you slowly start to back your way towards your ship.`
```

			choice

> 				`	Leave the planet immediately.`

Reviewer: ```suggestion
				`	(Leave the planet immediately.)`
```

					goto leave

> 				`	Distance yourself from the site but continue to watch.`

Reviewer: ```suggestion
				`	(Distance yourself from the site but continue to watch.)`
```

					goto watch

> 				`	Try to help the alien defeat the beast.`

Reviewer: ```suggestion
				`	(Try to help the alien defeat the beast.)`
```

> 					goto help
> 					

Reviewer: ```suggestion
```

			label help

> 			`	Despite being in a first contact situation without a proper interaction, you decide to attempt assisting the alien and aim your sidearm at the beast, preparing to fire. Instead of digging back under the surface, the beast continues to thrash about where it is while the alien batters it down with their spear weapon. Making sure you don't accidentally hit them, you hold your breath and wait until the right moment comes, aiming for the creature's head. Everything becomes tense as the space between you and the flailing creature diminishes, and your pulse begins to race. A gust of sand is forced directly towards you, and as it clears by that's when you fire your weapon. A laser shot flies directly into the face of the beast.`

Reviewer: ```suggestion
			`	Despite being in a first contact situation without a proper interaction, you decide to attempt assisting the alien and aim your sidearm at the beast, preparing to fire. Instead of digging back under the surface, the beast continues to thrash about where it is while the alien batters it down with their spear weapon. Making sure you don't accidentally hit them, you hold your breath and wait until the right moment comes, aiming for the creature's head. Everything becomes tense as the space between you and the flailing creature diminishes, and your pulse begins to race. A gust of sand blasts directly towards you, and once it clears you fire your weapon. A laser shot flies directly into the face of the beast.`
```

> 			`	Much to your dismay, the shot almost immediately ricochetes off of it, and you watch as the massive monster tumbles over you, just barely leaping into the air above. The organism plummets back into the ground behind you, and you fall to your back with the intensity of the dive with the wind knocked out of you. Raising your sidearm again, you aim once more, this time with your life possibly on the line. The creature explodes out of the ground once again with its gaping mouth pointed upward to the sky. With shaking hands, you take aim once again, and your finger slowly starts to pull on the trigger.`

Reviewer: ```suggestion
			`	Much to your dismay, the shot almost immediately ricochets off of it, and you watch as the massive monster tumbles over you, just barely leaping into the air above. The organism plummets back into the ground behind you, and you fall to your back with the wind knocked out of you. Raising your sidearm again, you aim once more, this time with your life possibly on the line. The creature explodes out of the ground once again with its gaping mouth pointed upward to the sky. With shaking hands, you take aim once again, and your finger slowly starts to pull on the trigger.`
```

			`	A very loud cannon-like bang follows, but it most certainly did not come from your weapon. The beast before you falls downward in a cloud of black smoke, blasting sand and rubble everywhere with a thud. Once you're able to come to your senses, you look up to see the alien marching out of the mouth of it and towards your direction, which has you stand up quickly in response.` 
			choice

> 				`Attempt to communicate with the alien.`

Reviewer: ```suggestion
				`(Attempt to communicate with the alien.)`
```

			`	At this point, you have already invested a great amount of your energy into the situation. With a last "thump" of their foot, the alien now stands before you, giving you a much better look at them. They are at least several feet taller than you, extremely bulky, and slightly hunched over. Heavy-looking metallics surround a majority of their body, and in any exposed areas are rough-looking patches of skin and hair. A great deal of facial hair covers their face, and above human-like eyes are a pair of horns curved inward towards you.`
			`	You aren't sure how to begin a conversation, but start by trying to greet the alien in a formal manner. Awkward silence ensues, and the alien looks you up and down in an almost unenthused manner. They emit a very deep grumble, and with a loud snort, reach into a crevice in their armor to grab something. Pulling out what looks to be some sort of compass or map, they point to a location on it, gesture to themselves, then lower the device slowly to you. It seems they want to be taken to one of their species' planets.`

> 			`	You hesitantly take the device and turn to make your way to your ship. There doesn't feel to be any emergency to the situation, but you still feel uncomfortable. Before heading out, you take one last glimpse of the enormous creature in the background. Its head is very reptilian, close to dragon-like, with numerous eyes and a mane of spikes. A few notably small appendiges are found at either end of it, almost like a salamander, but its surface is definitely scaley. The various signatures on your radar earlier must have been others of their kind. Glaring at you, then at the creature, the alien utters a single intelligible word in a very deep voice.`

Reviewer: ```suggestion
			`	You hesitantly take the device and turn to make your way to your ship. There doesn't feel to be any emergency to the situation, but you still feel uncomfortable. Before heading out, you take one last glimpse of the enormous creature in the background. Its head is very reptilian, close to dragon-like, with numerous eyes and a mane of spikes. A few notably small appendages are found at either end of it, almost like a salamander, but its surface is definitely scaly. The various signatures on your radar earlier must have been others of their kind. Glaring at you, then at the creature, the alien utters a single intelligible word in a very deep voice:`
```

			`	"Voordt."
				flee
			label watch
			`	Cautiously retreating towards your ship, you keep your eyes and ears peeled for the creature, wondering if and where it might appear again. As if to answer your inquiry, the ground begins to rumble again, and off to your left, sand and rubble erupt into the air as the beast twists and turns across the surface. Following a loud bellow, the creature screeches in short bursts, almost like chirps, except much more deeper than that of a bird. As some of the dust clears, you see the alien figure breathing intensely as they continue to battle the beast.`
			choice

> 				`	Help.`

Reviewer: ```suggestion
				`	(Help the alien.)`
```

					goto help

> 				`	Leave.`

Reviewer: ```suggestion
				`	(Leave.)`
```

					goto leave
			label leave
			`	With the situation beginning to spin out of your control, you quickly turn around and make for your ship, taking off as soon as you can. You wonder what will become of the alien figure as while they are most certainly a member of the species inhabiting the nearby systems, it doesn't seem like any help will be sent for them anytime soon.`
				depart
				
			label "second encounter"
			`Blah blah you met the Quarg already and are now here.`

			
			label "return encounter"
			`	You came back. What do you do?`
			choice

> 				`	Watch.`

Reviewer: ```suggestion
				`	(Watch.)`
```

					goto watch

> 				`	Help.`

Reviewer: ```suggestion
				`	(Help.)`
```

# Copyright (c) 2022 by Saugia
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A

> # PARTICULAR PURPOSE.  See the GNU General Public License for more details.

Reviewer 2: ```suggestion
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.
```

# Copyright (c) 2022 by Saugia
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

mission "Gegno Vi: First Contact"
	landing
	source "Var Oti"
	on offer
		conversation		
			branch "second encounter"
				has "Giaru Gegno: Quarg Contact: offered"
			branch "return encounter"
				has "Gegno Vi: First Contact: deferred"
				

> 			`As you descend towards the surface of this seemingly uninhabited world, your radar systems start to pick up several faint signatures below in every direction. None of the signatures change their behavior as you continue, but you decide to keep your ship a few thousand meters above the surface. You take the time to carefully look around, but there doesn't seem to be anything of note besides the cracked rock formations and valleys of sand.`

Author: As in, they don't chase your ship, run away from your ship, do something crazy due to your presence, etc., anything that would suggest an acknowledgement to your presence.

			`	Suddenly, your radar systems pick up a larger signature on the horizon, and you see an odd bulky rock-like structure. Once you get close enough, the structure begins to resemble ships you've seen flying around the nearby systems, although it is half sunken into the sand.`
			choice
				`	(Land near the ship and investigate.)`
					goto land
				`	(Continue to search the surface.)`
					goto search
			label land

> 			`	You carefully land your ship nearby. Most of the faint radar signatures from earlier have disappeared out of range. Exiting your ship in an environment suit and your sidearm on hand, you walk towards the alien ship.`

Author: As in, you got close enough to the surface that they spread out so far they weren't in your display anymore.

			label search
			`	As you fly onward, an erratic signature appears on the edge of your radar. A large arch-like structure becomes visible in the distance, but disappears shortly after. You set your ship down close to the signature and exit to investigate.`
			`	Much to your surprise, there is a large humanoid figure standing where the signature originated from. You were sure your radar scans showed no life form signature, but standing in front of you is a large, bulky being cladded in what looks like a form of metallic armor, wielding some type of spear-like weapon. The figure moves around attentively as if it is intently focused on its surroundings, but appears to take no interest in your presence. Unlike other aliens you have met in the galaxy, this encounter feels very odd in that there is no immediate interaction with you.`
			choice
				`	(Approach the alien.)`
				`	(Turn back to my ship.)`

> 			`	Suddenly, the ground starts to vibrate with increasing intensity. You take out your sidearm in a panic, but as you look back up at the alien figure, you notice that they too enter a defensive stance - but not towards you. A large maw bursts open in the sand to your left, and out of it comes a massive creature hundreds of times the size of you, covered in spikes and scales. You barely have time to react as it plummets onto the sand to your right, but you get a glimpse of the alien creature riding the beast as they impale it with their weapon.`

Author: :)

				depart
				
				
			label "second encounter"
			`As you descend towards the surface of this seemingly uninhabited world, your radar systems start to pick up several faint signatures below in every direction. None of the signatures change their behavior as you continue, but you decide to keep your ship a few thousand meters above the surface. You take the time to carefully look around, but there doesn't seem to be anything of note besides the cracked rock formations and valleys of sand.`
			`	Suddenly, your radar systems pick up a larger signature on the horizon, and you see an odd bulky rock-like structure. Once you get close enough, you recognize it as one of the Gegno's warships, although it is half sunken into the sand.`
			choice
				`	(Investiage the Gegno ship.)`
					goto investigate
				`	(Resume your search of the surface.)`
					goto resume
			label investigate

> 			`	You carefully land your ship nearby. Most of the faint radar signatures from earlier have disappeared out of range. Exiting your ship in an environment suit and your sidearm on hand, you walk towards the Gegno ship.`

Author: As above.

			choice
				`	(Go search for the alien.)`
					goto "search again"
				`	(Carry on.)`
					defer
					
			label "search again"
			`	With the coordinates of the alien vessel still in the history of your ship's computer, you set off to find it again. On your flight there, your radar suddenly picks up another ship approaching, and you look up to see the same type of ship you saw before flying at a higher altitude over you in the opposite direction. It must be another pilot belonging to the alien species.`
			choice
				`	(Attempt to contact the ship.)`
				`	(Evade the ship.)`
			`	The ship continues to fly off behind you until it disappears. It seems there is no concern to help the alien on the surface. Eventually, the familiar sunken vessel appears on the horizon and you begin to fly around the area in search for life signatures, hoping that the alien is still alive. Nothing comes up besides the faint signatures of the sand creatures at the edges of your radar. You decide to follow the signatures instead.`

> 			`	The signatures keep randomly disappearing and reappearing elsewhere, and appear to lead to only empty space. After checking a few of the signatures, an erratic signature appears on the edge of your radar, just like during your first visit. Approaching the location, you recognize the beast that is swinging about on the surface, and catch glimpses of the alien from before continuing to fight it.`

Author: I'll revisit this paragraph.

			choice
				`	(Assist the alien in defeating the creature.)`
					goto help
					
	on defer
		set "Gegno Vi: First Contact: deferred"

mission "Giaru Gegno: Quarg Contact"
	landing
	source "Giaru Gegno"
	to offer
		has "First Contact: Quarg: offered"
	on offer
		log "Factions" "Gegno" `The Gegno are a humanoid species to the galactic east of the core, split amongst themselves into branches. One branch is the Gegno Vi, who are spoken of to be strong and forceful brutes. The other is the Gegno Scin, mentioned to be progressers of research and technology. Apart from them is a complacent civilian population.`
		log `Learned from the Quarg that the Gegno are not interested in interaction with outsider species.`
		conversation

> 			"After jumping through several systems of many alien ships, a Quarg ring world is a sight of relief to you. From the looks of it, this is a fully complete ringworld, a true sight to behold. Trillions of Quarg probably inhabit this ring, but what's more interesting to your eye is that some other aliens are also present. They too are humanoid, and the most common figures you come across are either slightly above or below an average human's height. Their skin is a smooth, light gray, and adorned with different sorts of leather that remind you of medieval-style human clothing."

Author: Will revisit.

			`	You preemptively begin to stand up, but you are cut short when you see the second Quarg raise its arms outward in a gesture you never thought you'd see from a Quarg. "This must be for you an exhilarating day, young traverser. I strongly imagine this is a first for both of us." It is now obvious that this Quarg is not the same one from earlier, and you begin to see a few differences in their attire and figure. The upbeat rhythm to their speech is not something you would expect of a Quarg.`
			choice
				`	"Where am I?"`
			`	The Quarg sits down across from you, still gesturing their arms about, albeit slowly, in a strange matter. "You have found yourself in the dwelling of the mighty Gegno, who have rooted their hardy empire here in the great galactic east." You wonder if the Quarg's use of boastful adjectives is because of gaps in his translations, or if they are intentionally speaking of the Gegno in that way. Their voice still comes from the lower tones of Quarg speech, clashing with the excited pace they are speaking at.`
			choice
				`	"The Gegno?"`
					goto gegno

> 				`	"Are the ships I’ve seen these Gegno?"`

Reviewer 3: Idk why exactly but this feels weird to me. Maybe better like this?
```suggestion
				`	"Do the ships I’ve seen belong to these Gegno?"`
```
or smth of the like? Idk mb its just because I'm bad at english

Author: I believe I tried something like this before but Anarchist didn't like it, but I'll commit this anyway and we'll see what Derpy and others have to say about it soon.

			`Suddenly, your radar systems pick up a larger signature further up ahead on the horizon, and in the distance is an odd bulky rock-like structure. Once you get close enough, the structure begins to resemble ships you've seen flying around nearby systems, although it is half sunken into the sand.`
			choice
				`	(Land near the ship and investigate.)`
					goto land
				`	(Continue to search the surface.)`
					goto search
			label land
			`	Still keeping your guard up, you carefully land your ship nearby. Most of the faint radar signatures from earlier have disappeared out of range, so it seems there's nothing coming after you at the moment. Exiting your ship in an environment suit with your sidearm on hand, you walk towards the alien ship while remaining alert.`

> 			`	On closer inspection, the alien ship appears a bit larger than you thought it was - around the size of a Firebird. The dark-brown hull looks to be made from materials that you aren't familiar with, and it has several metallic-looking parts both hatched in crevices and protruding outward. Oddly enough, it seems perfectly intact, but the way it is angled into the sand suggests that the ship has been here for some time. There doesn't seem to be any activity around it, and you aren't picking up any life signs nearby. Leaving the site, you decide to head back to your ship and take another look around the area.`

Author: Gegno ships, especially the Vi, are slightly larger all-around than human counterparts since the Gegno themselves are larger beings. Kind of like the Sheragi, but not to that extent. I figured being up close to one would show this scaling a bit better than from a distance, but if it's negligible I can remove that description.

			label search
			`	Continuing on your journey, a vague radar signature similar to the ones you saw earlier, though more erratic, appears a bit further out. As you approach it, a large arch-like structure becomes visible in the distance, but disappears shortly after. You set your ship down close by and ready yourself as you exit it once again to investigate.`
			`	Much to your surprise, there is a large humanoid figure standing where the signature originated from. You were sure your radar scans showed no life form signature, but standing in front of you is a large, bulky being cladded in what looks like a form of metallic armor, wielding some type of spear-like weapon. The figure moves around attentively as if it is intently focused on its surroundings, but appears to take no interest in your presence. Unlike other aliens you have met in the galaxy, this encounter feels very odd in that there is no immediate interaction with you.`
			choice
				`	(Approach the alien.)`
				`	(Turn back to your ship.)`

> 			`	All of a sudden, the ground starts to vibrate, rapidly increasing in intensity with every second. You take out your sidearm in a panic, but as you look back up at the alien figure, you notice that they too enter a defensive stance - but not towards you. A large maw bursts open in the sand to your left, and out of it comes a massive creature covered in spikes and scales, hundreds of times the size of you. Taking aim with your sidearm, you barely have time to react as it plummets back into the sand over beyond your right, but you get a quick glimpse of the alien creature riding it down into the desert as they impale it with their weapon. Unsure of what comes next during the omnious silence, you slowly start to back your way towards your ship.`

Author: I'll take a look into this.

			choice
				`	(Leave the planet immediately.)`
					goto leave
				`	(Distance yourself from the site but continue to watch.)`
					goto watch
				`	(Try to help the alien defeat the beast.)`
			label help
			`	Despite being in a first contact situation without a proper interaction, you decide to attempt assisting the alien and aim your sidearm at the beast, preparing to fire. Instead of digging back under the surface, the beast continues to thrash about where it is while the alien batters it down with their spear weapon. Making sure you don't accidentally hit them, you hold your breath and wait until the right moment comes, aiming for the creature's head. Everything becomes tense as the space between you and the flailing creature diminishes, and your pulse begins to race. A gust of sand blasts directly towards you, and once it clears you fire your weapon. A laser shot flies directly into the face of the beast.`

> 			`	Much to your dismay, the shot almost immediately ricochets off of it, and you watch as the massive monster tumbles over you, just barely leaping into the air above. The organism plummets back into the ground behind you, and you fall to your back with the wind knocked out of you. Raising your sidearm again, you aim once more, this time with your life possibly on the line. The creature explodes out of the ground once again with its gaping mouth pointed upward to the sky. With shaking hands, you take aim once again, and your finger slowly starts to pull on the trigger.`

Author: Sounds good. I wasn't sure what the line was between consistency and repetition for the sake of a narrative.

			`	You hesitantly take the device and turn to make your way to your ship. There doesn't feel to be any emergency to the situation, but you still feel uncomfortable. Before heading out, you take one last glimpse of the enormous creature in the background. Its head is very reptilian, close to dragon-like, with numerous eyes and a mane of spikes. A few notably small appendages are found at either end of it, almost like a salamander, but its surface is definitely scaly. The various signatures on your radar earlier must have been others of their kind. Glaring at you, then at the creature, the alien utters a single intelligible word in a very deep voice:`
			`	"Voordt."`

> 				flee

Author: This part I still have yet to add. This is the last part of the intro mission I'm working on- we transport him, then talk to the Quarg, and that changes depending on whether or not you've met the Quarg already as well. I'll get to your other comments in a bit here- just skimming things at the moment because this review was a surprise to me :P

			`	On closer inspection, the ship appears a bit larger than you thought it was - around the size of a Firebird. The dark-brown hull looks to be made from materials that you aren't familiar with, and it has several metallic-looking parts both hatched in crevices and protruding outward. These characteristics seem to be representative of the Gegno Vi's brutish nature. Oddly enough, the warship seems perfectly intact, but the way it is angled into the sand suggests that the ship has been here for some time. There doesn't seem to be any activity around it, and you aren't picking up any life signs nearby. Leaving the site, you decide to head back to your ship and take another look around the area.`
			label resume
			`	Continuing on your journey, a vague radar signature similar to the ones you saw earlier, though more erratic, appears a bit further out. As you approach it, a large arch-like structure becomes visible in the distance, but disappears shortly after. You set your ship down close by and ready yourself as you exit it once again to investigate.`

> 			`	Much to your surprise, there is a large humanoid figure standing where the signature originated from. You were sure your radar scans showed no life form signature, but standing in front of you is a large, bulky being cladded in what looks like a form of metallic armor, wielding some type of spear-like weapon. This appearance is much more different than that of the Gegno you've seen on the nearby Quarg ringworld. Much like that of the crashed Gegno warship, their appearance suggests that they are a member of the Gegno Vi, perhaps some kind of warrior. The Vi moves around attentively as if it is intently focused on its surroundings, but appears to take no interest in your presence. Unlike other alien encounters you've had in the galaxy, the Vi does not approach you at all.`

Author: I don't think so. I think at points I felt referring to them as an it didn't seem appropriate, but if it's for the sake of consistency as mentioned before, I don't mind.

			`	After checking a few of the signatures, many of which would randomly disappear or reappear elsewhere, one of them strikes out to you as eccentric as the one from the first encounter. You hurriedly guide your ship to it and land, preparing yourself for whats to come. You've gone this far, so there's no point in turning around now. Approaching the location, you recognize the beast that is swinging about on the surface, and catch glimpses of the alien from before continuing to fight it.`
			choice
				`	(Assist the alien in defeating the creature.)`
					goto help
					
	on defer
		set "Gegno Vi: First Contact: deferred"

mission "Giaru Gegno: Quarg Contact"
	landing
	source "Giaru Gegno"
	to offer
		has "First Contact: Quarg: offered"
	on offer
		log "Factions" "Gegno" `The Gegno are a humanoid species to the galactic east of the core, split amongst themselves into branches. One branch is the Gegno Vi, who are spoken of to be strong and forceful brutes. The other is the Gegno Scin, mentioned to be progressers of research and technology. Apart from them is a complacent civilian population.`
		conversation

> 			"After jumping through several systems of many alien ships, a Quarg ring world is a sight of relief to you. From the looks of it, this is a fully complete ringworld, a true sight to behold. Trillions of Quarg probably inhabit this ring, but what's more interesting to your eye is that some other aliens are also present. They too are humanoid, and the most common figures you come across are either slightly above or below an average human's height. Their skin is a smooth, light gray, and adorned with different sorts of leather that remind you of medieval-style human clothing."

Author: Do you feel it's not necessary that we bring up the idea that the player went through a large alien territory before seeing a familiar sight? (Is this due to the small change the player doesn't meet the Quarg?)

			`	The mood here is eerie as the Quarg here move in a more quiet manner than what you're used to. They are almost dismissive of the other species walking about, who are equally as dismissive of you. The unusual atmosphere around you is finally broken when a few Quarg start to advance towards you, moving much quicker than the rest. After a moment of unidentifiable discussion and gestures between themselves, one of them motions you to follow them.`

> 			`	You walk with the Quarg, occasionally having trouble keeping pace with their silent yet brisk steps, as you head towards an area that appears devoid of the other aliens. Eventually, you are led to a smaller room with two more Quarg in waiting, silently sitting at the end of it. They talk, but this time you can hear a few familiar sounding terms, one being "Humani". Despite speaking to each other for several minutes, the Quarg still look at each other in what you assume is something like bewilderment.`

Author: I believe the Quarg over there spoke that, but if it's directly due to being Korath-derived, and not a Quarg word, then I will change it to something else for the Gegno Quarg.

				`	"Is it a problem that I'm here?"`
				`	(Do not say anything.)`
			`	The Quarg continues to look at you and bows its head slightly in an apologetic manner, as it seems neither they, nor any other Quarg on this ringworld, can speak your language. You continue to patiently wait in the uneasy silence, withholding your curiosities as almost half an hour goes by.`
			`	Finally, the silence is broken when the second Quarg from earlier enters the room again. This time, they glare at you much more energetically, and you begin to wonder if they are going to escort you off of the ring for being somewhere you shouldn't be.`
			`	You preemptively begin to stand up, but your action is caught short when you see the second Quarg raise its arms outward in a gesture you never thought you'd see from a Quarg. "Extravagant day, young traverser, this must be for you. A very substantial first this is for both of us, I strongly imagine." It is now obvious that this Quarg is not the same one from earlier, and you begin to see a few differences in their attire and figure. The upbeat rhythm to their speech is not something you would expect of a Quarg.`
			choice
				`	"Where am I?"`

> 			`	The Quarg sits down across from you, still gesturing their arms about, albeit slowly, in a strange matter. "You, brave rover, have found yourself in the dwelling of the mighty Gegno, who have rooted their hardy empire here in the great galactic east." You wonder if the Quarg's use of boastful adjectives is because of gaps in his translations, or if they are intentionally speaking of the Gegno in that way. Either way, they leave an odd impression on you- their voice still comes from the lower tones of Quarg speech, rather than something more excitable accompaying the pace that they are speaking at.`

Author: ~~Appreciate looking at this part, I'm not good with Quarg speech~~

			choice
				`	"The Gegno?"`
					goto gegno
				`	"Are the ships I’ve seen these Gegno?"`

> 					goto ships

Author: Will try to split it up more.

			label gegno

> 			`	"Yes, the imperious Gegno. A juvenile race, they are, many decades younger than that of you humans. They do not find themselves in the struggle of leadership the same way as others do. Rather, they glorify their pursuit of guidance between one another to share a noble future."`

Author: Will revisit this.

			choice 
				`	"What is this guidance they pursue?"`
				`	"A noble future?"`
					goto guidance
			label ships
			`	"Indeed, the vessels of such stature represent the robust Gegno. Not in sorrow, but greatness do they pursue their future, bearing the fruit of sacrifice as a trophy to their name." The Quarg now shifts facial expressions in such a way to suggest a small hint of frustration."`
			choice

> 				`	"Do the Gegno upset you in some way?"`

Author: This particular option was made so it also leads into the "frustrated" quarg viewpoint. Is that too empty of a transition?

			label guidance
			`	The Quarg now shifts facial expressions in such a way to suggest a small hint of frustration. "The Gegno believe it to be necessary to overwhelmingly sacrifice whatever must be, in order to imagine a grand future for themselves. Thus, they have separated into two different branches. One brandishes themselves as the honorable Vi - mightiest of warriors, toughest of kin. They are brutes that see proof of strength being the one and only variable in determining their path. That of the Scin, the more technologically able, believe that only through progression of sciences and discovery, will the Gegno prosper."`
				goto next
			label sacrifice
			`	"The Gegno believe it to be necessary to sacrifice whatever must be, in order to imagine a future for themselves. Thus, they have separated into two different branches. One brandishes themselves as the honorable Vi - mightiest of warriors, toughest of kin. They are brutes that see proof of strength being the one and only variable in determining their path. That of the Scin, the more technologically able, believe that only through progression of sciences and discovery, will the Gegno prosper."`

			label next

> 			choice

Author: It was kind of a way to give the player('s pilot) an option to choose which faction they seem to relate to more in terms of ideology, since a good portion of the story will be centered around exploring both sides' objectives. Is there a better way you suggest?

				`"The Vi are right, the strength to survive is more important than anything else." `
					goto vi
				`"The Scin are right, technological advancement opens up so many possibilities."`
					goto scin
				`"Why do they not combine their efforts to better their future together?"`
					goto together
			label vi
			`	"While it is characteristic, such a tremendous proposition is rather dull for a marvelously growing species. Be that as it may, you humans follow this harsh doctrine, though to a much lesser degree than that of the Vi."`
				goto end
			label scin
			`	The Quarg now shifts facial expressions in such a way to suggest a small hint of disappointment. "The pursuit of technology leaves behind many values important to a glorious species' growth, that of which we have seen plentifully."`
				goto end
			label together
			`	"Regrettably, most Gegno do not have a drive to pursue their principal future, diluting themselves far into their stagnant present. Segregation of followings was one of few ways to seek motivation beyond the fact."`
			label end
			choice
				`	"It seems that you do not like the paths they've chosen."`

> 			`	With its physique raising above you, the Quarg now stands up, darting its gaze off to the side. The verve it exhibited earlier is now long gone. "It is not in our prosperous grasp to persuade the Gegno, as once we did attempt to. They would ensure conflict with us should we interfere, and we have no desire to witness an imperile uprising of such a sort once more." It pauses, then gestures you to stand up as well. "Similarly, the Gegno do not wish to interact with audacious outsiders. Unfortunately, only a brief history of them we can tell. We cannot advocate virtuous pleasantries between you and them, though you are welcome at this ring should you find yourself here again."`

Author: noun
enthusiasm or vigor, as in literary or artistic work; spirit:
Her latest novel lacks verve.

Probably too rare of a word to use in this case. I'll find another synonym.

			label "trouble"
			`	"Quite a brave one you indeed are. Most would think to not leap into the daunting face of uncertainty as you have." The Quarg momentarily meditates on its next thought. "Though, as the Gegno have throughout their history, some may make monumentous jumps undeterred by any amount of hesitance."`
			`	The Quarg raises from its seat, and you act in accordance. "Please do be distinctly cautious in your encounters with the Gegno. Always feel gracious to visit here once more, should you mindfully chose to do so." As you are directed out of the room, you turn back to see the Quarg bowing. You can't help but wonder what was on their mind during the conversation, but it might be for the best you are more careful from now on.`
				decline

# Gegno Superheavy

mission "Stationed Augen"
	invisible
	landing
	npc save
		government "Gegno Vi"
		personality heroic vindictive staying
		system "Kiluit"
		ship "Augen" "Vytii Yrrlausei"
	to fail

> 		has "event: Vesvii war"

Reviewer 4: ```suggestion
		has "event: Vesvii war"
	to complete
		never
```

mission "Augen Destruction"
	landing
	invisible
	to offer
		has "Stationed Augen: failed"
		not "event: Vesvii war"
	on offer
		set "dead augen"
		fail

mission "Active Augen"
	invisible
	to offer
		has "event: Vesvii war"
		has "Stationed Augen: failed"
		not "dead augen"
	npc save
		government "Gegno Vi"
		personality heroic vindictive staying
		system "Vesvii"

> 		ship "Augen" "Vytii Yrrlausei"

Reviewer 4: ```suggestion
		ship "Augen" "Vytii Yrrlausei"
	to complete
		never
```

event "Vesvii War"

# Gegno Defense trigger

mission "Gegno Genocide Defense"
	landing

> 	repeat

Reviewer 4: ```suggestion
```

	invisible
	to offer

> 		not "Gegno Genocide Defense: active"

Reviewer 4: ```suggestion
```

		or
			"reputation: Gegno" < -100
			"reputation: Gegno Vi" < -100

> 			"reputation: Gegno Scin" < -100

Reviewer 4: The conversations suggests you're unwanted now because you attacked civvies, yet this can offer even if you never attacked the main Gegno government.

> 	on offer

Reviewer 4: ```suggestion
	to complete
		never
	on offer
```

		fail "Stationed Augen"
		"reputation: Gegno" = -100
		"reputation: Gegno Vi" = -100
		"reputation: Gegno Scin" = -100
		event "gegno: unified defense"
		conversation
			`As soon as you land your ship, you receive an incoming hail from a Quarg that sounds very displeased. "It was incredibly unwise to engage in the unnecessary destruction of noncombatants belonging to a species not of your own. A species that is still learning from themselves is vulnerable to outside influence, and your aggressive actions against civilians of the Gegno have resulted in needless events.`
			choice
				`	"It was an accident!"`
				`	"What are you talking about?"`
					goto "what"
				`	"I do not care."`
					goto "care"
			`	"It is greatly unlikely for violence upon unarmed victims to be accidental. Regardless of the cause, the effect has already occurred. The Gegno have banded together under one flag in an effort to deter any more outside interference. You are no longer welcome in their space, and you are advised to not visit again, lest we show you the same belligerent behavior you have exhibited to their citizens."`

> 					accept

Reviewer 4: ```suggestion
				accept
```

			label "what"

> 			`	"Your behavior has caused the Gegno to unite their fleets and deter any more presence from outsiders. It is a great shame. The time they would use to resolve their inner conflicts is now wasted upon hostility towards strangers due to your naivety. You are no longer welcome in Gegno space, and if you dare visit again, we shall demonstrate the same treatment you gave their citizens."`

Reviewer 4: This mission doesn't git negative Quarg rep, yet this Quarg is threatening me?

				accept
			label "care"

> 			`	The Quarg makes an unsightly sound. "Should you be seen in Gegno space once more, expect the same sentiment."`

Reviewer 4: "An unsightly sound"? A sound that isn't nice to look at?

		government "Gegno Vi"
		to spawn
			has "event: gegno: unified defense"
		to despawn
			has "dead augen"
		personality heroic vindictive staying
		system "Heutesl"
		ship "Augen" "Vytii Yrrlausei"

				
# Gegno Hunter Fleets

mission "Gegno Hunter Fleets"
	landing
	invisible
	repeat
	to offer
		has "Gegno Genocide Defense: offered"
		not "Gegno Hunter Fleets"
	npc
		government "Gegno"
		personality nemesis heroic vindictive
		system
			government "Gegno"
		fleet
			names "gegno united"
			variant 
				"Gneiss" 3
				"Hornfel" 2
				"Hornfel (Heavy Weapons)" 2
				"Eclogite" 5
				"Feldspar" 3
				"Halite" 18
				"Felsic" 3
				"Coesite" 5
	npc
		government "Gegno"
		personality nemesis heroic vindictive
		system
			government "Gegno"
		fleet
			names "gegno united"
			variant
				"Feldspar" 2
				"Halite" 12
				"Gneiss" 4
				"Felsic" 4
				"Coesite" 4
				"Eclogite" 3
	npc
		government "Gegno"
		personality nemesis heroic vindictive
		system
			government "Gegno"
		fleet
			names "gegno united"
			variant
				"Feldspar" 3
				"Halite" 18
				"Gneiss" 2
				"Hornfel" 3
				"Hornfel (Heavy Weapons)"
				"Eclogite" 3
				"Coesite" 6

# Gegno Vi Duels

mission "Gegno Vi Duels"

> 	minor

Author: So it doesn't interfere and bug out other missions being a passive mission. Even if it's invisible, repeat missions like this can possibly interrupt or create issues with others. This was discussed quite a bit and we decided this was best.
